containers/Ideas/IdeaCreate.js

```diff
@@ -134,13 +136,17 @@ export const IdeaCreate = props => {
               return `${acc}\n${curr}`
             }, 'Please try again.\n')
             setError(errorString)
+            setSubmitPending(false)
           } else {
             setError('Required fields are missing.')
+            setSubmitPending(false)
           }
         } else if (response.statusCode == 403) {
```

Comment:
just saw this (it was there before), which needs to be fixed:
because we unpack it some lines above we should use `statusCode === 403` instead of `response.statusCode == 403`.

And in Javascript we want to  compare strictly which is using `===`.